helm/vertical-pod-autoscaler-app/templates/crds/networkpolicy.yaml

```diff
@@ -14,19 +14,9 @@ spec:
   podSelector:
     matchLabels:
       job-name: {{ include "vertical-pod-autoscaler.crds.fullname" . }}
-  # allow egress traffic to the Kubernetes API
+  # allow egress traffic to everywhere
```

Comment:
Since this is a one-time job, it's just easier and does not introduce any meaningful security threat to have all egress whitelisted.